### DISCUSSION TITLE
Gcsfuse is very slow when compared to gsutil

### DISCUSSION THREAD

**Discussion by pratikkumar018** [GoogleCloudPlatform/gcsfuse#1535] [General]
Hi Team,
Our usecase is we are training models & we want to keep the training directory on cloud that is gcp buckets. 
There are two options. One is copying the model checkpoints through gsutil & second is mounting the bucket as a disk & directly storing there. 
We are more keen on the second option but we found that the second option is way too slower than the first option. 
Is there any particular reason for this.

**Comment by Tulsishah** [comment]
Hi @pratikkumar018, Thank you for reaching out. In the meantime, could you please provide the following details to help us better understand the situation:

1. Number of files involved.
2. Size of the files.
3. Whether writes(on different files) are occurring in parallel or sequentially. 
4. Whether writes are being performed on new or existing files.

Thanks, 
Tulsi Shah
